media-sound/pulsemixer/metadata.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>jchelmertt3@gmail.com</email>
+		<name>John Helmert III</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+</pkgmetadata>
```

Comment:
Since this a Github package, could you add an appropriate <upstream> section?
See https://devmanual.gentoo.org/ebuild-writing/misc-files/metadata/index.html.

I also would be happy, if you add an <longdescription> section, but it's not a must, if you don't want.

Reply:
`<upstream>` section added. I'd add a `<longdescription>` but I'm not sure what I could add that would be much better than the ebuild's `DESCRIPTION`.